### PR TITLE
Update WP Courses banners to WP Learn

### LINKED ISSUE #72682: Task: Update the WP Courses banner cards in the Calypso My Home
### Details

Currently on the My Home page of Calypso WP Courses is being promoted in two locations.

A banner in the top slider position:
Screenshot on 2023-01-24 at 14-15-42

And a banner in the slider further down where the videos are promoted:
Markup 2023-01-26 at 14 45 44

The top one definitely needs to be updated with a new image, copy, and link.

We have three image designs that can be used depending on the specs needed:
W_Learn@2x
W_Learn@2x
W_Learn_logo@2x
@garretsteider-a8c can assist with adjustments if needed.

Copy and link can be changed to:
World-class education by WordPress experts
Build your skills with access to webinars, courses, articles, support docs, a community and more! No enrollment required. No deadlines. Learn at your own pace.
Get started here [https://wordpress.com/learn/]

The second banner could be updated with the same information above, or potentially removed. Out of five cards in this position, three  are promoting videos, two are not. Personally, I'd recommend removing the two that are not promoting educational videos and keep the space focused on video content only (but that's not my decision to make).

### Checklist

- [ ] Swap image
- [ ] Swap copy and link

### Related

_No response_

### PR DESCRIPTION
**Pull request by obenland**
#### Proposed Changes

* Updates copy and assets of WP Courses task and card to reflect the switch to WP Learn.

Before | After
--- | ---
Screenshot 2023-01-26 at 2 41 33 PM | Screenshot 2023-01-26 at 2 16 45 PM
Screenshot 2023-01-26 at 2 41 48 PM | Screenshot 2023-01-26 at 2 36 27 PM

#### Testing Instructions

* Checkout this PR or use the Calypso.live link in the comments.
* Navigate to a new site's Home screen.
* Verify the links work as expected, the logos look good etc.

#### Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?

Fixes #72682

### CODE REVIEW

**Comment by obenland** [comment]
@tanjoymor The new heading is longer than the previous one, resulting in "experts" [widowing](https://www.indesignskills.com/tutorials/widows-and-orphans/). I added a non-breaking space to keep "WordPress expert" together. Not sure how big of a deal this is and/or whether you want to think of a shorter heading.

File: client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx [hunk 1]
```diff
@@ -5,10 +5,10 @@ import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
 const WPCourses = () => {
 	return (
 		<Task
-			title="Introducing WordPress.com Courses"
-			description="Build your skills with in-depth courses for any level, taught by WordPress experts. Enroll to access on-demand content, a private online community, office hours, and virtual meetups."
-			actionText="Enroll today!"
-			actionUrl="https://wpcourses.com/?utm_source=wordpressdotcom&utm_medium=referral&utm_campaign=courses_launch_myhome"
+			title="World-class education by WordPress&nbsp;experts"
```
Comment (p-jackson) [inline client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx:8]: We can `import { preventWidows } from 'calypso/lib/formatting'` to do the nbsp trick more semantically. And it means the nbsp won't get submitted to translators.

File: client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx [hunk 2]
```diff
@@ -5,10 +5,10 @@ import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
...
+			description="Build your skills with access to webinars, courses, articles, support docs, a community and more! No enrollment required. No deadlines. Learn at your own pace."
```
Comment (p-jackson) [inline client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx:9]: This copy wasn't being translated, which is a bug. If you want to take the opportunity to fix that now you can use the `useTranslate()` hook.

File: client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx [hunk 3]
```diff
@@ -5,10 +5,10 @@ import Task from 'calypso/my-sites/customer-home/cards/tasks/task';
...
+			actionText="Get started here"
+			actionUrl="https://wordpress.com/learn/?utm_source=wordpressdotcom&utm_medium=referral&utm_campaign=courses_launch_myhome"
```
Comment (p-jackson) [inline client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx]: Should the campaign slug change to `learn_launch_myhome`? 🤷‍♂️ 

```suggestion
			actionUrl="https://wordpress.com/learn/?utm_source=wordpressdotcom&utm_medium=referral&utm_campaign=learn_launch_myhome"
```

**Review by p-jackson — APPROVED** [review]
Good quick fix. I left some comments but this PR as it is addresses the urgent issue of deprecating (W) Courses.

We probably have a maintenance task to go through and rename all the identifiers from "course" to "learn"; including in the backend. Otherwise a few years from now no one will remember why it's sometimes called one thing and sometimes called another thing. I don't think there's a team currently assigned to maintain My Home though.

**Comment by tanjoymor** [comment]
> @tanjoymor The new heading is longer than the previous one, resulting in "experts" [widowing](https://www.indesignskills.com/tutorials/widows-and-orphans/). I added a non-breaking space to keep "WordPress expert" together. Not sure how big of a deal this is and/or whether you want to think of a shorter heading.

@obenland I’m okay with the non-breaking space you’ve used. I think it looks good. No idea if there are design guidelines around this? @autumnfjeld any thoughts?

File: client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx (hunk 1 above)
Comment (obenland) [reply client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx:8]: This won't get translated as the card is only available to English locales (https://github.com/Automattic/wp-calypso/pull/55684). I have a weak preference for the simplicity of adding it inline but happy to change if we generally prefer using `preventWidows`.

Thanks for adding the import to the suggestion, that would have saved a step for me!

File: client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx (hunk 2 above)
Comment (obenland) [reply client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx:9]: This card is only available to English locales (https://github.com/Automattic/wp-calypso/pull/55684)

File: client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx (hunk 1 above)
Comment (p-jackson) [reply client/my-sites/customer-home/cards/tasks/wp-courses/index.jsx:8]: Makes sense. Non-translated text doesn't come up often enough that I think we have a guideline about preferring `preventWidows`. I'm happy to leave the nbsp inline 👍

**Comment by obenland** [comment]
Will merge for now. If there's anything that needs changing we can do that in a follow-up PR

**Comment by autumnfjeld** [comment]
Sorry, coming late to the discussion. All looks fine to me. :)

**Pull request merged**